tests/Microsoft.Identity.Web.Test.Common/Mocks/MockHttpMessageHandler.cs

```diff
@@ -12,13 +13,13 @@ namespace Microsoft.Identity.Web.Test.Common.Mocks
 {
     public class MockHttpMessageHandler : HttpMessageHandler
     {
-        public Func<MockHttpMessageHandler, MockHttpMessageHandler> ReplaceMockHttpMessageHandler;
```

Comment:
The httpClient mock will auto-ignore instance discovery, since this is an MSAL implementation detail.